src/pages/Facility/FacilitiesPage.tsx

```diff
@@ -65,11 +65,15 @@ export function FacilitiesPage() {
     <div className="container px-4 py-8 mx-auto">
       <div className="flex items-start justify-between w-full">
         <Link href="/" className="">
-          <img src={mainLogo?.dark} alt="Care Logo" className="w-auto h-12" />
+          <img
```

Comment:
This component is used in two places - for logged in nearby facilities and logged out facilities public page.

So, this solution doesn't work as it hides the logo from public view.